docs/user-guide/src/ipam/ipam.md

```diff
@@ -0,0 +1,52 @@
+# IPAM (IP Address Manager)
+
+The IPAM project provides a controller to manage static IP address allocations
+in [Cluster API Provider Metal3](https://github.com/metal3-io/cluster-api-provider-metal3/).
+
+In CAMP3, the Network Data need to be passed to ironic through the BMH (Bare
+Metal Host). CAPI addresses the deployment of Kubernetes clusters and nodes, using the
+Kubernetes API. As such, it uses objects such as Machine Deployments (similar
+to deployments for pods) that takes care of creating the requested number of
+machines, based on templates. The replicas can be increased by the user,
+triggering the creation of new machines based on the provided templates.
```

Comment:
This part seems not directly related to the context, I would rather drop these lines and extend lines 12-14. You have mentioned that it is not possible to provide static IP addresses, but can we explain why?

Reply:
These lines are removed. However, I have problems describing the reason why the static IP addresses are not possible to be provided.

Reply:
Updated, please check.